ArduCopter/GCS_Mavlink.cpp

```diff
@@ -118,6 +118,49 @@ void GCS_MAVLINK_Copter::send_position_target_global_int()
         0.0f); // yaw_rate
 }
 
+void GCS_MAVLINK_Copter::send_position_target_local_ned()
+{
+    if (!copter.flightmode->in_guided_mode()) {
+        return;
+    }
+    
+    const GuidedMode guided_mode = copter.mode_guided.mode();
+    Vector3f target_pos;
+    Vector3f target_vel;
+    uint16_t type_mask;
+
+    if (guided_mode == Guided_WP) {
+        type_mask = 0x0FF8;
+        target_pos = copter.wp_nav->get_wp_destination();
+        target_vel = Vector3f(0.0f, 0.0f, 0.0f);
```

Comment:
These are implicitly zero; you don't need to do this call at all.

Reply:
Okay, thanks for the info. I'll clean this up. Anything else which I should do?

Reply:
Cleaned up the function, added comments also.
 
> Another thing which can be done is to convert into centimeters when we get the data itself in the appropriate condition
> 
> ```
> target_pos = copter.wp_nav->get_wp_destination() * 0.01f;
> ```

Should I add this also?

Reply:
Yes, definitely.  No unnecessary work, and sometime we might be clever about optimising that multiply.

Reply:
Done